Shared/Flows/Launch/LaunchCoordinator.swift

```diff
@@ -20,7 +20,7 @@ class LaunchCoordinator: PresentableCoordinator<LaunchStatus> {
         super.start()
 
         Task {
-            let launchStatus = await LaunchManager.shared.launchApp()
+            let launchStatus = await LaunchManager.shared.launchApp(with: self.deepLink)
```

Comment:
Doesn't really make a difference now, but now the launch manager actually uses the passed in deep link.